### PR TITLE
Set Versioned Integration Context - modified version arg in set versioned int context to be agnostic to type

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
https://github.com/demisto/etc/issues/28477

## Description
Modified the `set_integration_context` to be agnostic to type of value given to `version`

## Does it break backward compatibility?
   - [x] Yes
       - Changed func arg order

## Must have
- [x] Tests

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@Itay4 Review is done.

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -4055,7 +4055,7 @@ def merge_lists(original_list, updated_list, key):
     return merged_list
 
 
-def set_integration_context(context, sync=True, version=-1):
+def set_integration_context(context, version, sync=True):
```
Comment (glicht) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:4058]: Why do we need this order change? We can still leave version with a default of -1 just treat it properly in the code.

**Comment by meirwah** [comment]
@Itay4 what's the status here ?

**Comment by amit-davidson** [comment]
What's the status here?

**Review by glicht — APPROVED** [review]

**Pull request merged**